proxies/live/apiproxy/targets/sync-wrap.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<TargetEndpoint name="sync-wrap">
+  <PreFlow>
+    <Request>
+      <Step>
+        <Name>OauthV2.VerifyAccessToken</Name>
+      </Step>
+      <Step>
+        <Name>Quota</Name>
+      </Step>
+      <Step>
+        <Name>SpikeArrest</Name>
+      </Step>
+      <Step>
+        <Name>AssignMessage.AddSyncWaitHeader</Name>
+        <Condition>(request.header.X-Sync-Wait LesserThan 0.25) OR (request.header.X-Sync-Wait GreaterThan 29) OR (request.header.X-Sync-Wait Is null)</Condition>
+      </Step>
+    </Request>
+  </PreFlow>
+  <LocalTargetConnection>
+    <APIProxy>sync-wrap-{{ APIGEE_ENVIRONMENT }}</APIProxy>
+    <ProxyEndpoint>default</ProxyEndpoint>
+    <Path>/sync-wrap/{{ SERVICE_BASE_PATH }}</Path>
+  </LocalTargetConnection>
+  <FaultRules>
```

Comment:
Also need the `rate_limit_exceeded` Fault Rule from `ig3.xml` (minus the `AssignMessage.Mirror.*` steps)